rules/etcd-auto-tls-disabled/raw.rego

```diff
@@ -0,0 +1,31 @@
+package armo_builtins
+
+# Check if --auto-tls is not set to true
+deny[msga] {
+	etcdPod := input[_]
```

Comment:
Where do you validate that this is an etcd pod?

Reply:
validation has been added also inside rego rule as well